### PR TITLE
Trivial fixups in diff vs upstream

### PR DESCRIPTION
**Pull request by markmc**
Details in the individual commit messages

### CODE REVIEW

File: data/data/baremetal/main.tf [hunk 1]
```diff
@@ -20,7 +20,6 @@ module "bootstrap" {
 module "masters" {
   source = "./masters"
 
-  ironic_uri     = var.ironic_uri
```
Comment (stbenjam) [inline data/data/baremetal/main.tf:23]: Why is this being removed?

Comment (markmc) [reply data/data/baremetal/main.tf:23]: See https://github.com/openshift-metal3/kni-installer/pull/137/commits/ea152901d6a40052520727713078d2d05ac0e4de

```
baremetal: remove unused ironic_uri var from masters

This was used for the now deleted wait-for-ironic stuff.
```

Comment (stbenjam) [reply data/data/baremetal/main.tf:23]: Sorry, I read this entirely wrong, didn't realize it was under masters.  Thanks.

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/848/

**Pull request merged**